src/start/cp2k_runs.F

```diff
@@ -5,7 +5,7 @@
 
 ! **************************************************************************************************
 MODULE cp2k_runs
-   USE dbcsr_acc_device,                ONLY: acc_get_ndevices,&
+   USE dbcsr_acc_device,                ONLY: dbcsr_acc_get_ndevices,&
```

Comment:
This is wrong. It is forbidden to use dbcsr functions not in the API. Please use dbcsr_api for that